bin/start_do.sh

```diff
@@ -336,10 +336,11 @@ pulumi $pulumi_args up
 header "Bank of Sirius"
 cd "${script_dir}/../pulumi/python/kubernetes/applications/sirius"
 pulumi $pulumi_args up
-THE_FQDN=$(pulumi config get kic-helm:fqdn -C ${script_dir}/../pulumi/python/config)
-THE_IP=$(kubectl  get service kic_nginx_ingress  --namespace nginx-ingress --output=jsonpath='{.status.loadBalancer.ingress[*].ip}')
 
 header "Finished!"
+THE_FQDN=$(pulumi config get kic-helm:fqdn -C ${script_dir}/../pulumi/python/config || echo "Cannot Retrieve")
+THE_IP=$(kubectl get service kic-nginx-ingress  --namespace nginx-ingress --output=jsonpath='{.status.loadBalancer.ingress[*].ip}' || echo "Cannot Retrieve")
```

Comment:
Another bash question: I know that the `||` will stop the script from crashing if the command exits nonzero, but what's the difference between `echo` and just doing `|| "Cannot Retrieve"`?  I assume the goal is to set `THE_IP` and `THE_FQDN` to "Cannot Retrieve"?

Reply:
We're running the script in a mode that it exits if there are any failures anywhere (anything return non-zero) - you can see the set statement at the top.

If you look at the CRD removal logic you can see one approach to potentially having errors and wanting to ignore them -we turn off the strict abort on error and allow them to error.

In this case, I wanted to keep going but I also wanted to put a message out there. So what happens is if the RC from CMD1 is non-zero, we run CMD2. If CMD2 exits non-zero it's an error, but if it's 0 then we go. Basically, "if you are unable to get this info substitute this phrase". The other construct you'll see a lot is CMD1 && CMD2 which means "if CMD1 exits 0, then run CMD2".